app/models/reservation.rb

```diff
@@ -120,6 +120,9 @@ def self.upcoming_offline(start_at_limit)
       .where(product_id: OfflineReservation.current.pluck(:product_id))
       .not_canceled
       .not_ended
+      .joins(:order_detail)
+      .where("order_details.state" => %w(new inprocess))
+      .where("order_details.problem" => false)
```

Comment:
Were you able to confirm how the job that sends reservations to the problem queue works? And does it ignore cart reservations?